Jint.Tests.Test262/Test262Test.cs

```diff
@@ -67,20 +66,42 @@ protected void RunTestCode(string code, bool strict)
             }
 
             string lastError = null;
-            try
-            {
-                engine.Execute(code);
-            }
-            catch (JavaScriptException j)
+
+            bool negative = code.IndexOf("negative:", StringComparison.Ordinal) > -1;
+            if (negative)
             {
-                lastError = TypeConverter.ToString(j.Error);
+                try
+                {
+                    engine.Execute(code);
+                }
+                catch (JavaScriptException j)
+                {
+                    lastError = TypeConverter.ToString(j.Error);
+                }
+                catch (Exception e)
+                {
+                    lastError = e.ToString();
+                }
+
+                Assert.NotNull(lastError);
             }
-            catch (Exception e)
+            else
             {
-                lastError = e.ToString();
-            }
+                try
+                {
+                    engine.Execute(code);
+                }
+                catch (JavaScriptException j)
+                {
+                    lastError = TypeConverter.ToString(j.Error);
+                }
+                catch (Exception e)
+                {
+                    lastError = e.ToString();
+                }
 
-            Assert.Null(lastError);
```

Comment:
Shouldn't the `negative` check be around this line? The rest is identical in both conditions

Reply:
Indeed, you caught me copy-pasting, shame on me. Changed to only check varying part.